package.json

```diff
@@ -29,7 +29,7 @@
         "@openpgp/asmcrypto.js": "^2.3.2",
         "@openpgp/tweetnacl": "^1.0.3",
         "@openpgp/web-stream-tools": "^0.0.10",
-        "openpgp": "npm:@protontech/openpgp@~5.3.0"
+        "openpgp": "github:openpgpjs/openpgpjs#main"
```

Comment:
Is this required for this change?

Reply:
Ah sorry, didn't read the PR description. Yeah, if it's not critical, maybe let's wait for that?